docs/code.rst

```diff
@@ -132,6 +132,9 @@ Variable                            Description
 ``{{ latest_date }}``               same as ``{{ ds }}``
 ``{{ ti }}``                        same as ``{{ task_instance }}``
 ``{{ params }}``                    a reference to the user-defined params dictionary
+``{{ var.value }}``                 global defined variables represented as a dictionary
+``{{ var.json }}``                  global defined variables represented as a dictionary
```

Comment:
Could you just update the code.rst doc to be more explicit about how to get the variables? Show something like:

    {{ var.value.my_var }}
   {{ var.json.my var}}

Can also mention that JSON dictionaries are supported:

   {{ var.json.my dict_var.key}}